Core/Contributions/Solutions Software/EditableListView.cls

```diff
@@ -475,28 +469,6 @@ itemAndColumnAt: aPoint
 		ifTrue: [itemIndex@columnIndex]
 		ifFalse: [nil]!
 
-itemIndex: anInteger hasFocus: aBoolean
```

Comment:
`itemIndexWithFocus[:]` was an (inefficient) duplicate of `caretIndex[:]`, and `itemIndex:hasFocus:` misunderstands the nature of the LVIS_FOCUSED style—though it appears to be a per-item style, the control enforces that it may only be applied to one item at a time.

Fair point, though, that these may be public API—I added them back with `#deprecated` tagging, aliasing `itemIndexWithFocus[:]` to `caretIndex[:]` and explaining why `itemIndex:hasFocus:` does not allow any behavior that simply setting the `caretIndex` does not.